_pages/jobs.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: page
+permalink: /jobs/
+title: "Jobs &amp; Opportunties"
+page-title: "Jobs &amp; Opportunties"
+breadcrumb: "contribute"
+---
+
+# Jobs &amp; Opportunties
+
+There are many ways to work with M-Lab, whether as a direct hire, through a fellowship or internship; as a contracor, collaborator, or researcher at another institution; or as an open source contributor. Here we list open opportunities to work for or with M-Lab directly, and also opportunities within other organizations with whom we collaborate.
```

Comment:
Can we add a quick prompt here that if folks have open jobs they want us to link out to, to let us know?

Reply:
sure thing.